modules/ballerina-test/src/test/java/org/ballerinalang/test/statements/continuestatement/ContinueStmtTest.java

```diff
@@ -15,14 +15,12 @@
 * specific language governing permissions and limitations
 * under the License.
 */
-package org.ballerinalang.model.statements;
+package org.ballerinalang.test.statements.continuestatement;
```

Comment:
Shall we move this to package org.ballerinalang.test.statements.continue? Change the resource bal path accordingly.

Reply:
We cannot use `continue` in a package path since it is a keyword in java.